canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.jsx

```diff
@@ -181,20 +194,20 @@ class EditorForm extends React.Component {
 						</div>
 					);
 				} else {
-					tabContent.push(
+					tabContentAcc.push(
 						<div key={this._getContainerIndex(hasAlertsTab, i) + "-" + key}
 							className={classNames("properties-category-container", { "properties-hidden-container": tab.content.itemType === ItemType.TEARSHEET })}
 						>
-							<button type="button" onClick={this._showCategoryPanel.bind(this, tab.group)}
-								className={classNames("properties-category-title", { "properties-light-enabled": this.props.controller.getLight() }) }
+							<AccordionItem title={`${tab.text}${this._getMessageCountForCategory(tab) ? this._getMessageCountForCategory(tab) : ""}`}
+								// Open Tab with Alert Message when from Alerts Tab or a open Default Tab
+								open={ this.defaultOpenTab === tab.group || this.alertOpenTab === tab.group }
+								onHeadingClick={this._showCategoryPanel.bind(this, tab.group)}
+								className={`bx--accordion__item-${i} ${classNames("properties-category-content",
```

Comment:
Why are we using `bx--accordion__item-${i}` for a classname?  This seems odd.

Reply:
I have removed this classname now for the content.